data/backend/authentication/forms.py

```diff
@@ -0,0 +1,76 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.forms import ModelForm
+from django.utils.translation import gettext_lazy as _
+from mailauth.backends import MailAuthBackend
+from mailauth.forms import EmailLoginForm as MailauthEmailLoginForm
+import json
+import os
+import sys
+
+
+class EmailLoginForm(MailauthEmailLoginForm):
+    def get_login_url(self, request, token, next=None):
+        return super(EmailLoginForm, self).get_login_url(
+            request, token, next=next or "/#/auth/loggedIn"
```

Comment:
`/#/auth/loggedIn` is in camelCase while all other Elm routes are in lowercase separated by hyphens, so for consistency this should be `logged-in` (and I suggest `authenticated` instead)